### ISSUE TITLE
Sync the vendored CSS Validator with the fix for `w3c/css-validator#506`

### ISSUE THREAD

**Issue by maxi07** [validator/validator#2115]
### Summary

The CSS Validator sources vendored in `validator/validator` do not include the fix for [`w3c/css-validator#506`](https://github.com/w3c/css-validator/issues/506), which was merged upstream as [`w3c/css-validator@610d484`](https://github.com/w3c/css-validator/commit/610d484c0bcfeaf3d37661b4406b4debe69b6268).

The fix is absent from the vendored [`RGBA.java`](https://github.com/validator/validator/blob/cf9eb402c4957d2404644951d3f46d8330d59db4/css-validator/org/w3c/css/values/color/RGBA.java) at `validator/validator@cf9eb402`. Consequently, a directly built `vnu.jar` still fails on this minimal stylesheet when it is linked from an HTML document.

Could you sync the vendored `css-validator` sources with an upstream revision that includes `610d484` and the subsequent custom-property handling from [`w3c/css-validator#511`](https://github.com/w3c/css-validator/pull/511)?

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>CSS custom property validation</title>
  <style>
    :root {
      --rgb: 13, 110, 253;
    }

    a {
      color: rgba(var(--rgb), 0.5);
    }
  </style>
</head>
<body>
  <a href="#">Test</a>
</body>
</html>
```

### Checker error messages

```shell
CSS: “color”: Cannot invoke "org.w3c.css.values.CssValue.getType()" because "val" is null.
```

### Steps to reproduce

```sh
java -jar build/dist/vnu.jar --errors-only repro.html
```

### Expected behavior

No error.

### Actual behavior

See error message.

**Comment by sideshowbarker** [comment]
Updated to the latest css-validator sources and pushed to the W3C checker:

https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bbase64%2CPCFkb2N0eXBlIGh0bWw%2BCjxodG1sIGxhbmc9ImVuIj4KPGhlYWQ%2BCiAgPG1ldGEgY2hhcnNldD0idXRmLTgiPgogIDx0aXRsZT5DU1MgY3VzdG9tIHByb3BlcnR5IHZhbGlkYXRpb248L3RpdGxlPgogIDxzdHlsZT4KICAgIDpyb290IHsKICAgICAgLS1yZ2I6IDEzLCAxMTAsIDI1MzsKICAgIH0KICAgIGEgewogICAgICBjb2xvcjogcmdiYSh2YXIoLS1yZ2IpLCAwLjUpOwogICAgfQogIDwvc3R5bGU%2BCjwvaGVhZD4KPGJvZHk%2BCiAgPGEgaHJlZj0iIyI%2BVGVzdDwvYT4KPC9ib2R5Pgo8L2h0bWw%2BCg%3D%3D
